### PR TITLE
v4.8.3

### PR DESCRIPTION
**Pull request by chongjunkim**
# Please ensure that all information is filled out to streamline the approval of your pull request.

## Pre-Release
*Create a pre-release and include the link to the tagged repo*
- ```add link here```

## Functionality Added?
- ```add bullet points with functionality added```
- ``example: Added ability for spider webs to shoot out of the tool.``

## Pull Request Checklist

- [ ] Updated README.md?
  - **README.md** - add instructions on how to use your new functionality, if applicable. ```/docs/```

- [ ] Updated Meta Script?

  - Is this a **Production** Release?
    - **meta.js & meta.user.js** - Updated all instance of the version numbers in both files.

      ``
      preRelease branch -> /assets/js/
      ``
    <br>

  - Is this a **Pre-Release** Release?
    - **preRelease.user.js** - Updated all instance of the version numbers in both files.

      ``
      develop branch -> /assets/js/
      ``

- [ ] Updated Change Log

  - All updates related to the end user (visual & functionality) will require an update to the change log.

    - **Changelog.md** - add bullet points on what was added.

      ``
      /docs/
      ``

### CODE REVIEW

File: launchToolbar/js/launchToolbar.js [hunk 1]
```diff
@@ -1329,6 +1355,76 @@ function programVariables() {
...
   };
+	
+	//jQuery -Jun
+	$(document).ready(function () {
+		$(window).scroll(function (event) {
+			if ($(window).scrollTop() > 115)
+				$('#uiBox').css({
+					'position' : 'fixed',
+					'top' : 'auto',
+//					'width' : '99%',
```
Comment (cirept) [inline launchToolbar/js/launchToolbar.js]: please remove all comments from the source code.   =]

File: launchToolbar/js/launchToolbar.js [hunk 2]
```diff
@@ -1329,6 +1355,76 @@ function programVariables() {
...
+					'display': 'inherit',
+					'width': 'inherit',
+				});
+				$('#sidebarDiv, #handlebarContainer .indicator, #handlebarContainer #handle').css({
+					'margin-top' : '60px',
+				});
+			}
+		});
+		
+//		if ($('.sidebarCell, #sidebarDiv').width() > 100) {
```
Comment (cirept) [inline launchToolbar/js/launchToolbar.js]: please remove =]

File: launchToolbar/js/launchToolbar_meta_prerelease.user.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+// ==UserScript==
```
Comment (cirept) [inline launchToolbar/js/launchToolbar_meta_prerelease.user.js]: this is perfect!  thank you for doing this.

File: launchToolbar/js/launchToolbar_meta.user.js [hunk 4]
```diff
@@ -3,14 +3,14 @@
...
+// @version 4.8.3
 // @run-at document-end
-// @downloadURL https://raw.githubusercontent.com/cirept/salesforceToolbar/master/launchToolbar/js/launchToolbar_meta.user.js
+// @downloadURL https://github.com/cirept/salesforceToolbar/raw/master/launchToolbar/js/launchToolbar_meta.user.js
 // @require https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js
 // @resource fontAwe https://use.fontawesome.com/3953f47d82.js
 // @resource googFont https://fonts.googleapis.com/css?family=Raleway:600,900
 // @resource jqueryUI https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/themes/smoothness/jquery-ui.css
-// @require https://raw.githubusercontent.com/cirept/salesforceToolbar/v4.8.2/launchToolbar\js\launchToolbar.min.js
+// @require https://github.com/cirept/salesforceToolbar/raw/master/launchToolbar/js/launchToolbar.min.js
```
Comment (cirept) [inline launchToolbar/js/launchToolbar_meta.user.js]: This links to the .min.js file.  I don't believe you have minified your file, so please link it to the main launchToolbar.js file.

File: launchToolbar/js/launchToolbar.js [hunk 5]
```diff
@@ -1329,6 +1355,76 @@ function programVariables() {
...
+				}),
+				$('#sc').css({
+					'position' : 'fixed',
+					'top' : '35px',
+				});
+			else
+				$('#uiBox').css({
+					'position' : 'absolute',
+					'top' : '115px',
+//					'width' : '98.5%',
```
Comment (cirept) [inline launchToolbar/js/launchToolbar.js]: please remove comments

File: launchToolbar/js/launchToolbar.js [hunk 6]
```diff
@@ -80,31 +80,35 @@ function programVariables() {
             'id': 'uiBox',
           })
           .css({
-            'position': 'fixed',
+            'position': 'absolute',
             'display': 'none',
             'z-index': '9999',
-            'background': 'linear-gradient(to bottom, rgb(192, 229, 248), #F6F6F6)',
+//            'background': 'linear-gradient(to bottom, rgb(192, 229, 248), #F6F6F6)',
```
Comment (cirept) [inline launchToolbar/js/launchToolbar.js]: please remove comments =]

File: launchToolbar/js/launchToolbar.js [hunk 7]
```diff
@@ -69,8 +69,8 @@ function programVariables() {
       launchToolbar.config = {
         '$placeholder': jQuery('<div>')
           .css({
-            'height': '50px',
-            'display': 'none',
+//            'height': '0',
```
Comment (cirept) [inline launchToolbar/js/launchToolbar.js]: please remove commented out code

**Review by cirept — DISMISSED** [review]
Please update the commented portions of the code.

File: launchToolbar/js/launchToolbar_meta_prerelease.user.js (hunk 3 above)
Comment (chongjk88) [reply launchToolbar/js/launchToolbar_meta_prerelease.user.js]: No problem :D

**Comment by chongjunkim** [comment]
Modified everything that was requested. :)

**Review by cirept — APPROVED** [review]

**Pull request merged**